packages/shared/styles/components/molecules/SfBanner.scss

```diff
@@ -2,80 +2,82 @@
 
 .sf-banner {
   box-sizing: border-box;
-  display: flex;
-  min-height: var(--banner-min-height, 9rem);
-  padding: var(--banner-padding, var(--spacer-big));
-  color: var(--banner-color, var(--c-text));
-  background-repeat: no-repeat;
-  background-position: var(--banner-background-position, bottom left);
-  background-size: cover;
-  background-color: var(--banner-background);
+  display: var(--banner-display, flex);
+  flex-direction: var(--banner-flex-direction, column);
+  justify-content: var(--banner-justify-content, left);
+  background-color: var(--background-color);
+  background-position: var(--banner-background-position, center);
+  background-repeat: var(--banner-background-repeat, no-repeat);
+  background-size: var(--banner-background-size, cover);
   background-image: var(--background-image);
+  color: var(--banner-color, var(--c-text));
+  width: var(--banner-size, var(--banner-width, 100%));
```

Comment:
I think it should be `var(--banner-width)`. We use sizes if there is the same dimension for both properties, wdyt ? @pspaczek @AdamPawlinski

Reply:
Ok. I remove that.